app/models/conversation.rb

```diff
@@ -57,6 +58,7 @@ class Conversation < ApplicationRecord
   before_validation :validate_additional_attributes
   validates :additional_attributes, jsonb_attributes_length: true
   validates :custom_attributes, jsonb_attributes_length: true
+  validates :uuid, uniqueness: true
```

Comment:
Is this intended to be required to be unique?  I opened this PR in part to understand that.

Reply:
Or, should it be unique within the context of a single account?

Reply:
@jordan-brough  this looks good to me. The idea is for the UUID to be unique across the installation since we do this look-up. 

https://github.com/chatwoot/chatwoot/blob/f7d4f39b5c8efb6e1f423b867c8c2845aabc73eb/app/mailboxes/reply_mailbox.rb#L47